internal/controller/entra/main.go

```diff
@@ -184,38 +196,56 @@ func (c *Controller) sendInviteNewEntraUser(ctx context.Context, email string, s
 		message = "You have been invited to join the UCL ARC Services Portal by " + string(sponsor.Username)
 	}
 	messageInfo := graphmodels.NewInvitedUserMessageInfo()
-
 	messageInfo.SetCustomizedMessageBody(&message)
 	requestBody.SetInvitedUserMessageInfo(messageInfo)
 
-	_, err := c.client.Invitations().Post(ctx, requestBody, nil)
+	sponsorData, err := c.client.Users().ByUserId(string(sponsor.Username)).Get(ctx,
+		&graphusers.UserItemRequestBuilderGetRequestConfiguration{
+			QueryParameters: &graphusers.UserItemRequestBuilderGetQueryParameters{
+				Select: []string{"id"},
+			},
+		},
+	)
```

Comment:
could be `c.userData`, maybe it should, probably is should

Reply:
https://github.com/ucl-arc-tre/portal/pull/337/commits/3cecc310a10149951ae4d2d2d9cc18202dcb6325